pkg/arvo/app/dojo.hoon

```diff
@@ -663,6 +679,15 @@
           :-  i=""
           t=(turn `wain`?~(r.hit ~ (to-wain:format q.u.r.hit)) trip)
       ==
+    ::
+    ++  dy-inspect
+      |=  [topics=(list term) sut=type]
+      %+  dy-rash  %mor
+      =+  to-display=(find-item-in-type:dprint (flop topics) sut)
+      ?~  to-display
+        [%tan [%leaf "Could not find help"]~]~
+      [(print-item:dprint u.to-display)]
```

Comment:
There's no way to look inside of unnamed cores that aren't built by an arm - which will be all of them at release. I need some kind of syntax to look inside of an unnamed core - maybe just `$` for now

`/foo/hoon`:
```hoon
|%
::  +bar: a bar
++  bar  ~
--
```
then
```
=a -build-file %/foo/hoon
# a$+bar
```

Reply:
going to be revisiting this now that we're not naming cores in the first release